dts/bindings/clock/st,stm32f0-pll-clock.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2021, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Main PLL node binding for STM32F0 and STM32F3 devices.
+
+  Takes one of clk_hse or clk_hsi as input clock.
+
+  Note: Some STM32F0 SoC variants (STM32072 for instance) may also support HSI48
+  as input clock. This case is not supported yet.
+
+  Frequency calculation for output clock is as follows:
+
+    f(PLLCLK) = f(PLLIN) x PLLMUL   --> SYSCLK (System Clock)
+
+    with, depending on the case:
+            f(PLLIN) = f(input_clk) / 2       if input_clk = clk_hsi
+            f(PLLIN) = f(input_clk) / PREDIV  if input_clk = clk_hse
+
+  Note: On STM32F070x6, STM32F070xB and STM32F030xC, PREDIV also applies when
```

Comment:
STM32F072 is not in your list. Looking at Reference Manual, concerned devices are STM32F04x, STM32F07x and STM32F09x.
Maybe easier to read like this:

f(PLLIN) = f(input_clk) / 2 if input_clk = clk_hsi              (STM32F03x and STM32F05x)
f(PLLIN) = f(input_clk) / PREDIV if input_clk = clk_hsi    (STM32F070x6, STM32F070xB and STM32F030xC)
f(PLLIN) = f(input_clk) / PREDIV if input_clk = clk_hse

Reply:
Arg, I took the wrong table... Fixing it

Reply:
> f(PLLIN) = f(input_clk) / 2 if input_clk = clk_hsi (STM32F03x and STM32F05x)
f(PLLIN) = f(input_clk) / PREDIV if input_clk = clk_hsi (STM32F070x6, STM32F070xB and STM32F030xC)
f(PLLIN) = f(input_clk) / PREDIV if input_clk = clk_hse

This doesn't take F3 into account